09.mesh-modeler/02.item-editor/docs.md

```diff
@@ -0,0 +1,90 @@
+
+**Grid Horizontal Size** If greater than 0, the item will be snapped to the intersections of an invisible horizontal grid, and the distance between the parallel lines of this grid will be this *Grid Horizontal Size* (in meters). If 0, there will be no horizontal snapping, you can place the item exactly where you want (but it will be difficult to align several items or align items with blocks).
+
+
+### Advanced placement parameters
+
+**Switch pivot manually** If activated, the only way to change the pivot to use will be the 'Q' key ('A' with azerty keyboard). If deactivated, the map editor will automatically change the pivot according to the direction of the surface you are aiming with the cursor (for example it may select the lowest pivot if you are aiming at the floor, and the highest pivot if you are aiming at a ceiling). If the editor selects several pivot candidates, you can cycle through them with the 'Q' key.
+
+**Fly Offset** Offsets vertically the invisible horizontal planes on which the cursor moves when it is in the air. For example, if the *Fly Step* parameter is 4 meters and the the *Fly Offset* is 0 meter, when you scroll with the mouse wheel, the cursor will move to the coordinates y = 0, 4, 8, 12, 16... m. But if you set the *Fly Offset* to 1 meter, then the cursor will be placed at y = 1, 5, 9, 13, 17... m. This parameter has no effect if *Fly Step* is 0.
+
+**Grid Horizontal Offset** It is recommended to leave this parameter to 0, unless you really know what you are doing. It offsets the horizontal grid by this amount of meters, in both horizontal axis (x and z). The only value that might be useful sometimes would be half of *Grid Horizontal Size*, but even in that case it is probably a better idea to divide *Grid Horizontal Size* by 2 instead.
+
+**Grid Vertical Size** If greater than 0, the item will be snapped on the vertical axis, and the value is the step of this snapping in meters. In other words, it sets the snapping along the y axis, just as the *Grid Horizontal Size* sets the snapping along the x and z axis. The difference with *Fly Step* is subtle: unlike *Fly Step*, *Grid Vertical Size* does not enable the item to fly. For instance, it can be used for an item which is meant to be placed on walls, to limit the available heights and then make it easy to align them. But if your item's *Fly Step* and *Grid Vertical Size* are both greater than 0, it is recommended to set the same value on both parameters.
+
+**Grid Vertical Offset** No effect if *Grid Vertical Size* equals 0. Otherwise, it offsets the available heights by this value in meters. For example, if *Grid Vertical Size* is 8 and *Grid Vertical Offset* is 2, the available heights are y = 2, 10, 18, 26, 34... m. If the item's *Fly Offset* is different from zero, it is recommended to put the same value in *Grid Vertical Offset*.
```

Comment:
```suggestion
**Grid Vertical Offset** No effect if *Grid Vertical Size* equals 0. Otherwise, it offsets the available heights by this value in meters. For example, if *Grid Vertical Size* is 8 and *Grid Vertical Offset* is 2, the available heights are `y = 2, 10, 18, 26, 34... m`. If the item's *Fly Offset* is different from zero, it is recommended to put the same value in *Grid Vertical Offset*.
```